cypress/support/fragments/eholdings/eHoldingsTitles.js

```diff
@@ -28,9 +28,9 @@ export default {
       eHoldingsTitle.waitLoading(title);
     });
   },
-  create: (packageName, titleName = `title_${getRandomPostfix()}`) => {
-    cy.do(resultSection.find(Button('New')).click());
-    eHoldingsNewCustomTitle.waitLoading();
+  create: (packageName, titleName = `title_${getRandomPostfix()}`, paneTitle) => {
+    cy.do(resultSection.find(Button({ href: '/eholdings/titles/new' })).click());
```

Comment:
Is this change required? In some cases it's preferable to use previous aproach

Reply:
@dnovikau I just changed it back to not overcomplicate things. But I had to delete eHoldingsNewCustomTitle.waitLoading() step from the method completely since inside it checks name of the Pain (in English), but in flow on the test this title won't be in English. So that was the point of previous overcomplications, so the question what is better to have: deleted waitLoading step or one more optional parameter with name of the Pane?

Reply:
From localization teststing perspective approach with locator using attribute values seams more convenient. Please leave proposed realization:
```JS
cy.do(resultSection.find(Button({ href: '/eholdings/titles/new' })).click());
```